Protecs/src/main/java/uk/ac/ucl/protecs/objects/Person.java

```diff
@@ -140,8 +140,18 @@ public static OCCUPATION getValue(String x) {
 	boolean recovered = false;
 	boolean hasCovid = false;
 	boolean hadCovid = false;
+	
+	// Covid testing properties
```

Comment:
as a placeholder; we discussed modularising and abstracting diseases here, especially in light of beginning to expand the set of diseases we're considering.